apps/chrome-devtools/src/app-devtools/theming-panel/common.ts

```diff
@@ -0,0 +1,26 @@
+import type { StylingVariables } from '@o3r/styling';
+
+export type Variable = StylingVariables extends (infer T)[] ? T : never;
+
+export const varRegExp = /^var\(--([^,)]*).*\)$/;
```

Comment:
You may need to handle `spaces` as well
```suggestion
export const varRegExp = /^var\(--([^, )]*).*\)$/;
```

(same comment in `searchFn` regexp)